Modules/xASL_module_ASL.m

```diff
@@ -949,12 +949,22 @@
     end
 end
 
-%% 2. Manage bUseBasilQuantification parameter that activates BASIL quantification
-if ~isfield(x.modules.asl, 'bUseBasilQuantification') || isempty(x.modules.asl.bUseBasilQuantification)
-	x.modules.asl.bUseBasilQuantification = false;
+%% 2. Manage bUseExternalQuantification parameter that activates External quantification quantification
+if ~isfield(x.modules.asl, 'bUseExternalQuantification') || isempty(x.modules.asl.bUseExternalQuantification)
+	if ~isfield(x.modules.asl, 'ExternalQuantificationType') || ~isempty(regexpi(x.modules.asl.ExternalQuantificationType, '(basil|fabber|vaby)'))
```

Comment:
Should we also test for:

if x.modules.asl.bUseExternalQuantification && isempty(x.modules.asl.ExternalQuantificationType)?

and 

if x.modules.asl.bUseExternalQuantification && isempty(regexpi(x.modules.asl.ExternalQuantificationType, '(basil|fabber|vaby)')?

BTW we want the regexp to include ^ $ to avoid the possibility for basil_fabber

Reply:
Empty Type is tested at the start of the population module.

we test already for having an unknown quantification type as described in a comment above.